src/main/java/seedu/ccacommander/model/util/SampleDataUtil.java

```diff
@@ -61,8 +61,8 @@ public static Event[] getSampleEvents() {
         };
     }
 
-    public static ReadOnlyAddressBook getSampleAddressBook() {
-        AddressBook sampleAb = new AddressBook();
+    public static ReadOnlyCcaCommander getSampleCcaCommander() {
+        CcaCommander sampleAb = new CcaCommander();
```

Comment:
Should this be sampleCcaCommander?
```suggestion
        CcaCommander sampleCcaCommander = new CcaCommander();
```

Reply:
resolved